### DISCUSSION TITLE
I love this!

### DISCUSSION THREAD

**Discussion by CoderAtGithub** [TriliumNext/Trilium#1358] [Ideas]
It would be amazing, if we can pitch in some money for an app :'(((!!!! Cloud syncing, honestly this would definitely beat notion... but this is a great note taking app! , also maybe the ability to (if its not there), to export as pdf, whether its one notebook, or a notebook with its nestings.. or multiple notebooks with their nestings...

But the app idea, thats like icing on the cake! :)

also it would be nice to have the syntax for python or any other coding language highlighted so that if someone is taking a note, they wouldnt have to nest to a new page in order to type their code. This would be ideal for someone who is learning coding and wants the syntax and what the instructor saying on the same page.

also some of the other note taking apps, when you export as pdf, the syntax that was highlighted in code does not show up, just shows up as black text, but in cherry tree thats not the case, if we can solve these, or make these, this would be one SOLIIIIDDDDD app (not that it isnt :) but you get the idea im sure , also im not a coder, im pretty sure its hard to even just add an idea just like that )

**Comment by avinashkanaujiya** [comment]
@BugBountyX Coding highlighting within a text note is a feature that ckeditor devs can do. It is not in zadam's hand. At least this is what he said when somebody asked for it in past.

**Reply by zadam** [reply]
You're right. I remember seeing some discussions on that topic in ckeditor issues and having highlighting inside ckeditor would be pretty difficult and would essentially mean embedding something like code mirror inside ckeditor. So I wouldn't bet on this anytime soon.

**Reply by good-praxis** [reply]
For reference issues discussion this include #196 and #1326

**Reply by CoderAtGithub** [reply]
hmm, what if we got together as a community and raised funds for it?

**Reply by meichthys** [reply]
@BugBountyX  Have you tried the mobile view? It works well for basic reference. 
The main issue i have with the mobile view is the lack of a search/jump feature. That makes it difficult to use with large note trees.
